locale/en_US/countries.xml

```diff
@@ -62,15 +62,15 @@
 	<country name="Colombia" code="CO"/>
 	<country name="Comoros" code="KM"/>
 	<country name="Congo" code="CG"/>
-	<country name="Congo, The Democratic Republic of the" code="CD"/>
+	<country name="Congo (the Democratic Republic of the)" code="CD"/>
 	<country name="Cook Islands" code="CK"/>
 	<country name="Costa Rica" code="CR"/>
 	<country name="Côte d'Ivoire" code="CI"/>
 	<country name="Croatia" code="HR"/>
 	<country name="Cuba" code="CU"/>
 	<country name="Curaçao" code="CW"/>
 	<country name="Cyprus" code="CY"/>
-	<country name="Czech Republic" code="CZ"/>
+	<country name="Czechia" code="CZ"/>
```

Comment:
Still think it would be nice to have an easy update-lists-from-ISO process…like isoquery-to-XML...